### ISSUE TITLE
Migrations conflict

### ISSUE THREAD

**Issue by NotUnlikeTheWaves** [iiacoban42/DelfiTLM#38]
When I try to boot up the application with `docker-compose up --build` I get the following error:

```
delfitlm-app-1      | Traceback (most recent call last):
delfitlm-app-1      |   File "/app/manage.py", line 23, in <module>
delfitlm-app-1      |     main()
delfitlm-app-1      |   File "/app/manage.py", line 19, in main
delfitlm-app-1      |     execute_from_command_line(sys.argv)
delfitlm-app-1      |   File "/usr/local/lib/python3.9/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
delfitlm-app-1      |     utility.execute()
delfitlm-app-1      |   File "/usr/local/lib/python3.9/site-packages/django/core/management/__init__.py", line 413, in execute
delfitlm-app-1      |     self.fetch_command(subcommand).run_from_argv(self.argv)
delfitlm-app-1      |   File "/usr/local/lib/python3.9/site-packages/django/core/management/base.py", line 354, in run_from_argv
delfitlm-app-1      |     self.execute(*args, **cmd_options)
delfitlm-app-1      |   File "/usr/local/lib/python3.9/site-packages/django/core/management/base.py", line 398, in execute
delfitlm-app-1      |     output = self.handle(*args, **options)
delfitlm-app-1      |   File "/usr/local/lib/python3.9/site-packages/django/core/management/base.py", line 89, in wrapped
delfitlm-app-1      |     res = handle_func(*args, **kwargs)
delfitlm-app-1      |   File "/usr/local/lib/python3.9/site-packages/django/core/management/commands/migrate.py", line 95, in handle
delfitlm-app-1      |     executor.loader.check_consistent_history(connection)
delfitlm-app-1      |   File "/usr/local/lib/python3.9/site-packages/django/db/migrations/loader.py", line 306, in check_consistent_history
delfitlm-app-1      |     raise InconsistentMigrationHistory(
delfitlm-app-1      | django.db.migrations.exceptions.InconsistentMigrationHistory: Migration delfin3xt.0001_initial is applied before its dependency ewilgs.0001_initial on database 'default'.
```

I'm as of yet unsure how to reconcile this but thought I'd note it if anyone has ideas on it

**Comment by iiacoban42** [comment]
I was stuck on this for too long... Comment delfin3xt in installed_apps form settings.py, then run the migration again 
![image](https://user-images.githubusercontent.com/43474282/145728199-c1a6e4bd-52ca-4c49-a6d0-b2f2f662f626.png)

**Comment by NotUnlikeTheWaves** [comment]
While that does work I'm not sure I think it's a good solution, as it means that the migrations in the repo currently are in an invalid state

**Comment by iiacoban42** [comment]
I think it's because applied the migrations for DelfiN3xt beforehand. And with #26 we introduced a relation between DelfiN3xt and Donwlink tables that causes this conflict.

**Comment by NotUnlikeTheWaves** [comment]
Ah I see. Apparently I still had an old database instance that might've been involved in the issue. Now that I've opened a fresh one it works just fine, I will close the issue

**Comment by StefanoSperetta** [comment]
This problem is always present every time we re-deploy a new instance on an empty database. The issue is caused by a known [bug](https://docs.djangoproject.com/en/4.0/releases/3.2.9/) in Django 3.8.2.

**Comment by iiacoban42** [comment]
Closed by #71
